pkg/proxy/iptables_portalmanager.go

```diff
@@ -0,0 +1,273 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
```

Comment:
New files should say 2015. You don't need to update the header in any files that you are modifying.